.krew.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  # 'name' must match the filename of the manifest. The name defines how
+  # the plugin is invoked, for example: `kubectl restart`
+  name: aws-auth
+spec:
+  # 'version' is a valid semantic version string (see semver.org)
+  version: {{ .Tag }}
+  # 'homepage' usually links to the GitHub repository of the plugin
+  homepage: https://github.com/keiko/aws-auth
+  # 'shortDescription' explains what the plugin does in only a few words
+  shortDescription: "Makes the management of the aws-auth config map for EKS Kubernetes clusters easier"
+  description: |
+    Useful for automation purposes, any workflow that needs 
+    to grant IAM access to an EKS cluster can use this plugin 
+    to modify the config map.
+  # 'platforms' specify installation methods for various platforms (os/arch)
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        platform: amd64
+    {{addURIAndSha https://github.com/keikoproj/aws-auth/releases/download/{{ .ProjectName }}_{{ .Tag }}_{{ .Os }}_{{ .Arch}}.tar.gz" .Tag}}
```

Comment:
I dont think .Os, .Arch and .ProjectName will be available here.. you might have to specify the filenames directly.
.Tag is OK

Not sure if it's possible to test this is working on your fork, but would be good to test prior to merging that we can use this from kubectl and everything works

Reply:
Actually Tag should be TagName

So it should be something like 
```
    {{addURIAndSha "https://github.com/keikoproj/aws-auth/releases/download/{{ .TagName }}/aws-auth_{{ .TagName }}_darwin_amd64.tar.gz" .TagName }}
```

so that it corresponds with release artifact `https://github.com/keikoproj/aws-auth/releases/download/v0.3.3/aws-auth_v0.3.3_darwin_amd64.tar.gz`

Reply:
Ok, I wasn't sure since the other PR was using `.Tag`. I can change it to `.TagName`